__tests__/app.test.js

```diff
@@ -52,6 +52,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: "11"
```

Comment:
Rather than editing a previous test, could we have a new test for this ticket please? You may need to change this one so it still passes, by using a more flexible matcher. 👍

Reply:
Separated comment_count into new test ready for review.